pkg/gui/filetree/file_node.go

```diff
@@ -42,19 +43,15 @@ func (s *FileNode) GetPath() string {
 }
 
 func (s *FileNode) GetChildren() []INode {
```

Comment:
in a follow up PR I'm going to merge commit_file_node and file_node: they were originally separate due to lack of generics